nebullvm/operations/optimizations/compilers/onnxruntime.py

```diff
@@ -22,14 +22,11 @@ class ONNXCompiler(Compiler):
         "cpu": [
             None,
             QuantizationType.STATIC,
-            QuantizationType.HALF,
             QuantizationType.DYNAMIC,
         ],
         "gpu": [
             None,
-            QuantizationType.STATIC,
```

Comment:
since we added TensorRT Provider, shouldn't this be supported?

Reply:
It needs further work because tensorrt needs a different calibration implementation, so for now it's disabled